client/logclient_test.go

```diff
@@ -176,9 +176,9 @@ func TestGetEntriesErrors(t *testing.T) {
 		{start: 1, end: 2, rsp: "", wantErr: "EOF"},
 		{start: 0, end: -1, wantErr: "end should be >= 0"},
 		{start: 3, end: 2, wantErr: "start should be <= end"},
-		{start: 1, end: 2, rsp: "not-json", wantErr: "invalid"},
-		{start: 1, end: 2, rsp: `{"entries":[{"leaf_input":"bogus","extra_data":"bogus"}]}`, wantErr: "illegal base64"},
-		{start: 1, end: 2, rsp: `{"entries":[{"leaf_input":"bbbb","extra_data":"bbbb"}]}`, wantErr: "failed to unmarshal"},
+		{start: 4, end: 5, rsp: "not-json", wantErr: "invalid"},
+		{start: 5, end: 6, rsp: `{"entries":[{"leaf_input":"bogus","extra_data":"bogus"}]}`, wantErr: "illegal base64"},
```

Comment:
Not sure why these start/end values changed?  Was that to trigger some different behaviours in the test?

Reply:
I changed it so every test case uses different values, so it's easier to spot which one failed. I could have added a description field...but I didn't.